test/test_transforms_tensor.py

```diff
@@ -728,6 +585,132 @@ def test_center_crop(device):
         scripted_fn.save(os.path.join(tmp_dir, "t_center_crop.pt"))
 
 
+@pytest.mark.parametrize('device', cpu_and_gpu())
+@pytest.mark.parametrize('func, method, out_length', [
+    # test_five_crop
+    ("five_crop", "FiveCrop", 5),
+    # test_ten_crop
+    ("ten_crop", "TenCrop", 10)
```

Comment:
let's get rid of those ugly `getattr` (not your fault!) and directly use F.five_crop, T.FiveCrop, etc.

Reply:
yep, nice call!